### ISSUE TITLE
Image loading in dataloader code

### ISSUE THREAD

**Issue by jshtok** [microsoft/i-Code#108]
Hello, and thank you very much for the contribution of the code.
While running your I have noticed that only the first page of a PDF file is loaded (as image). Indeed, in your class PregeneratedDatasetBase, the add_images() routine features the line 
` im = convert_from_path(im_path)[0]`
While in the original DUE code the _get_page_img() routine uses the page_no field to fetch the relevant page.

Can you please explain this situation?
Thank you!

**Comment by Coobiw** [comment]
Hello, I also busy with it! I'm curious about how can we get the images and corresponding Q-A pairs. Do you have any experience?

**Comment by jshtok** [comment]
Well, I just took the DUE repo as reference, and brute force fixed the
dataloader to fetch the relevant page.

**Comment by Coobiw** [comment]
Thanks for your reply! So I thought that you first generate the `memmaps`. After that, you use the memmaps to build the dataloader, and save the results(image-QA pairs) into a format file just like json?

**Comment by jshtok** [comment]
Yes, I used the DUE-baselines repo to generate the memmaps, then did
the finetuning, and then the --evaluate option in the train config saves
the predictions. Please notice you need to convert these predictions to
another format (back in the   DUE-baselines) in order to run them against
the GT annotations in the DUE-evaluator repo. Then, finally, you get the
performance numbers.

**Comment by Coobiw** [comment]
Thank you very much!!! I want convert the memmaps into png files because I want to use the images as inputs. I want to ask that what process repo did you use? The benchmarker in UDOP repo or the original one in DUEBenchmark/baselines?
